### ISSUE TITLE
[MASSEMBLY-871] Descriptor ref "bin" causes error messages in the build log on Windows

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-assembly-plugin#1079]
**[Stefan Ferstl](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=st.ferstl@gmail.com)** opened **[MASSEMBLY-871](https://issues.apache.org/jira/browse/MASSEMBLY-871?redirect=false)** and commented

When using the builtin assembly descriptor `bin`, the build output contains ERRORs when run on Windows:

```
[INFO] --- maven-assembly-plugin:3.1.0:single (create-bin-distro) @ descriptor-ref-demo ---
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative-reference (starting with slash) /
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative-reference (starting with slash) /
[INFO] Building tar: C:\path\to\my\project\target\descriptor-ref-demo-1.0.0-SNAPSHOT.tar.gz
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative-reference (starting with slash) /
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative-reference (starting with slash) /
[INFO] Building tar: C:\path\to\my\project\target\descriptor-ref-demo-1.0.0-SNAPSHOT.tar.bz2
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative-reference (starting with slash) /
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative-reference (starting with slash) /
```

However, the build succeeds and produces valid archives (tar.bz2, tar.gz and zip).

The problem should rather be logged as WARN than as ERROR.

To reproduce this issue, I created a demo project on GitHub: https://github.com/ferstl/MASSEMBLY-871 .

---

**Affects:** 3.1.0, 3.1.1

**Issue Links:**
- [MASSEMBLY-746](https://issues.apache.org/jira/browse/MASSEMBLY-746) Warnings about platform dependent paths inconsistent.

**Remote Links:**
- [GitHub Pull Request #10
](https://github.com/apache/maven-assembly-plugin/pull/10)

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

I can reproduce it. How would you solve this issue?

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Pushed a branch with a possible fix. Please validate. Passes on Windows. Need to see have it behaves on non-Windows. I have just noticed that this anti-pattern is used in many ITs.

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Pushed an updated branch. This should do. [Karl Heinz Marbaise](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=khmarbaise), does this look good to you?

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [9293ae87f8998f99261c2df516656661916973db](https://gitbox.apache.org/repos/asf?p=maven-assembly-plugin.git;a=commit;h=9293ae87f8998f99261c2df516656661916973db).
